### PR TITLE
Fixed getProjectPath with new ShowDirectory option

### PR DESCRIPTION
**Pull request by felipecaputo**

### CODE REVIEW

**Comment by felipecaputo** [comment]
@KyrychukD This was the fix, but unit tests I'll need to do later

**Comment by KyrychukD** [comment]
Cool, thanks for letting me know!

This looks way mode complicated than it should. `getProjectPath` could be just `return pickedObj.dir` if you just put dir there when creating the list of items.

File: package.json [hunk 1]
```diff
@@ -109,9 +109,9 @@
                     "description": "Define if extension should look for for SVN projects too."
                 },
                 "gitProjectManager.displayProjectPath": {
-                  "type": "boolean",
-                  "default": false,
-                  "description": "Indicates if project path will be displayed in project picker."
+                    "type": "boolean",
```
Comment (KyrychukD) [inline package.json]: Hmm, `.editorconfig` says indent size is 2.

**Comment by KyrychukD** [comment]
I am pretty sure you can add any info you want like this: https://github.com/KyrychukD/git-project-manager/commit/ac6303b2d753c163c39eece76ee99ba0a97db16b
as the signature suggest:

```
showQuickPick<T extends QuickPickItem>(items: T[] | Thenable<T[]>, options: QuickPickOptions & {canPickMany: true}, token?: CancellationToken): Thenable<T[] | undefined>
```
https://code.visualstudio.com/docs/extensionAPI/vscode-api#2045

**Comment by felipecaputo** [comment]
That's true!!!

I wasn't aware about the T **extends** QuickPickItem, so I can add more data then the QuickPick item

Man, thanks a lot!!!

**Comment by felipecaputo** [comment]
@KyrychukD will you open a PR with this change please?

**Comment by KyrychukD** [comment]
No problem, will open a PR in a minute! :)

**Comment by felipecaputo** [comment]
There is another small issue that I think I can take a look and the I release a 1.7.1 tomorrow

**Pull request merged**